### PR TITLE
docs: krew is wrong word use kubectl instead

### PR DESCRIPTION
**Pull request by subhamkrai**
krew is itself a plugin and it is wrong wording
to use krew plugin intead it should be kubectl
pluging.

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: Documentation/Troubleshooting/kubectl-plugin.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ See the [kubectl-rook-ceph documentation](https://github.com/rook/kubectl-rook-c
 
 ## Installation
 
-* Install [Krew](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)
+* Install [kubectl](https://kubectl.sigs.k8s.io/docs/user-guide/setup/install/)
```
Comment (travisn) [inline Documentation/Troubleshooting/kubectl-plugin.md:17]: Need to revert the change to the URL?
```suggestion
* Install [kubectl](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)
```

**Review by travisn — CHANGES_REQUESTED** [review]

File: Documentation/Troubleshooting/kubectl-plugin.md (hunk 1 above)
Comment (subhamkrai) [reply Documentation/Troubleshooting/kubectl-plugin.md:17]: yess, reverted

**Review by travisn — APPROVED** [review]
